src/plugins/unified_search/public/saved_query_management/saved_query_management_list.tsx

```diff
@@ -227,96 +323,179 @@ export function SavedQueryManagementList({
-                isVirtualized: true,
-              }}
-              renderOption={renderOption}
-            >
-              {(list, search) => (
-                <>
-                  <EuiPanel style={{ paddingBottom: 0 }} color="transparent" paddingSize="s">
-                    {search}
-                  </EuiPanel>
-                  {list}
-                </>
-              )}
-            </EuiSelectable>
-          </div>
-        </>
-      ) : (
-        <>
-          <EuiText
-            size="s"
-            color="subdued"
-            className="kbnSavedQueryManagement__text"
-            data-test-subj="saved-query-management-empty"
+              ),
+              onChange: debouncedSetSearchTerm,
+              'data-test-subj': 'saved-query-management-search-input',
+            }}
+            loadingMessage={i18n.translate(
+              'unifiedSearch.search.searchBar.savedQueryLoadingQueriesText',
+              {
+                defaultMessage: 'Loading queries',
+              }
+            )}
+            emptyMessage={
+              <span data-test-subj="saved-query-management-empty">
+                {noSavedQueriesDescriptionText}
+              </span>
+            }
+            onChange={(choices) => {
+              const choice = choices.find(({ checked }) => checked);
+              if (choice) {
+                handleSelect(
+                  currentPageQueries.find((savedQuery) => savedQuery.id === choice.value)
+                );
+              }
+            }}
+            renderOption={renderOption}
+            css={{
+              '.euiSelectableList__list': {
+                WebkitMaskImage: 'unset',
+                maskImage: 'unset',
+              },
+              '.euiSelectableListItem': {
+                color: 'inherit !important',
+                backgroundColor: 'inherit !important',
+              },
+            }}
           >
-            <p>{noSavedQueriesDescriptionText}</p>
-          </EuiText>
-        </>
-      )}
-      <EuiPopoverFooter paddingSize="s">
-        <EuiFlexGroup gutterSize="s" direction="column">
+            {(list, search) => (
+              <>
+                <EuiPanel color="transparent" paddingSize="s" css={{ paddingBottom: 0 }}>
+                  {search}
+                </EuiPanel>
+                <EuiPanel color="transparent" paddingSize="s">
+                  <EuiText size="xs" color="subdued">
+                    {countDisplay}
+                  </EuiText>
+                </EuiPanel>
+                <EuiHorizontalRule margin="none" />
+                {list}
+              </>
+            )}
+          </EuiSelectable>
+        </EuiFlexItem>
+        {totalQueryCount > SAVED_QUERY_PAGE_SIZE && (
+          <EuiFlexItem grow={false} css={{ padding: euiThemeVars.euiSizeS }}>
+            <EuiFlexGroup responsive={false} justifyContent="center">
+              <EuiFlexItem grow={false}>
+                <EuiPagination
```

Comment:
There is an UX issue that going to another page resets the selection:

![Jan-24-2024 11-49-45](https://github.com/elastic/kibana/assets/1415710/e9da58d2-1ac7-4fcf-a2ca-b18b6dfbb159)

Reply:
Ok sounds good 👍 I can update it to not clear the selection on page change.

About clicking the text to go to the page, it seems like it could be helpful from UX perspective, but I think it would be hard to implement since it would need to account for how the current search term impacts the pages as well as if additional queries are added/removed in the background (not actually sure how we could do this tbh).

One option might be to replace `1 selected` with something that indicates which query is selected either inline or on hover, like `selected: My saved query ti...` (and allow hover for full title). Not sure if this would look bad or not though, or how necessary it would really be in practice. We could also instead choose to just remove or keep the selected text for now, and consider how to improve things as a small followup PR.

Reply:
@davismcphee +1 we can take care of improvements in a follow-up PR. Let's keep the "1 selected" text for now then. Thank you.

Reply:
Sounds good! And I updated it to no longer clear the selected query on page change here: https://github.com/elastic/kibana/pull/170599/commits/02d7d6012c1622e1a7ffd11e26b4c935c790b377.